### PR TITLE
Add customizable batch_size for cleanup_activitystream and cleanup_jobs

### PR DESCRIPTION
**Pull request by dangoncalves**
##### SUMMARY
Add an option in cleanup_activitystream cli tool in order to customize batch size. This can be useful when default size (set to 500) failed.

##### ISSUE TYPE
  - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - CLI
 - 
##### AWX VERSION
```
awx: 23.0.1.dev11+g217c76d0b7
```

### CODE REVIEW

**Comment by AlanCoding** [comment]
I think this is a great idea, but we unfortunately have special logic to make it usable when launched from the API. This would logically need some additions (although I'd rather refactor it to work more generally)

https://github.com/ansible/awx/blob/e072bb7668d1565ac77af38a65ee937c0b3f303a/awx/main/tasks/jobs.py#L1876

**Comment by fosterseth** [comment]
@dangoncalves 

Would you be interested in adding "batch_size" as a value to pass in via the API? It would only be a couple extra lines of code, where @AlanCoding pointed out. That would made this additional parameter even more useful!

**Comment by AlanCoding** [comment]
also see https://github.com/ansible/awx/pull/13717

**Comment by dangoncalves** [comment]
@AlanCoding @fosterseth : I just updated my PR.

File: awx/main/tasks/jobs.py
Comment (AlanCoding) [inline awx/main/tasks/jobs.py]:
```suggestion
                    args.extend(['--batch-size', str(json_vars['batch_size'])])
```

By this point you already established that "batch_size" is in the vars, so there's no need to process default values in this location.

Comment (dangoncalves) [reply awx/main/tasks/jobs.py]: Thanks for code review.

File: awx/main/tasks/jobs.py
Comment (fosterseth) [inline awx/main/tasks/jobs.py:1876]: we want to move under the

```python
if system_job.job_type == 'cleanup_jobs':
```

block, because `batch-size` is not an argument to the cleanup_activitystream system job

**Comment by fosterseth** [comment]
@AlanCoding do you have ideas for test coverage around this?

**Comment by dangoncalves** [comment]
Hello @AlanCoding,

Just a gentle reminder: we are waiting for your answer :)

**Comment by AlanCoding** [comment]
Thanks. Could you do the DCO signoff before we merge?

https://github.com/ansible/awx/blob/159dd62d84df3ac14d274571724eee0324c2ca61/DCO_1_1.md?plain=1#L4

**Comment by dangoncalves** [comment]
Done

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**
